bootstrap.sh

```
@@ -1,10 +1,30 @@
 #!/bin/bash
+
```

Add robust shell options and prerequisite/tool checks near the top of the script. Right now the script can continue after partial failures. Consider adding `set -euo pipefail` immediately after the shebang and verifying required commands are present (`kind`, `kubectl`) and that `cluster.yml` exists. This satisfies the robustness requirement from the task (bootstrap.sh must run reliably and fail fast with clear messages).